core/src/main/java/org/apache/iceberg/MetadataUpdateParser.java

```diff
@@ -52,6 +52,7 @@ private MetadataUpdateParser() {
   static final String ADD_SNAPSHOT = "add-snapshot";
   static final String REMOVE_SNAPSHOTS = "remove-snapshots";
   static final String SET_SNAPSHOT_REF = "set-snapshot-ref";
+  static final String REMOVE_SNAPSHOT_REF = "remove-snapshot-ref";
```

Comment:
Nit: Can you move this up to be above `set-snapshot-ref`? This follows the order that the classes are declared in. So move this to line 54 (and let `set-snapshot-ref` go down one line).